UPGRADING.md

```
@@ -37,6 +37,18 @@ The `datastructures` (sub)library has been removed from the MQT Core repository.
 Its functionality has only ever been used in [MQT QMAP] since its inception.
 As a consequence, the code shall be moved to [MQT QMAP] once QMAP adopts an MQT Core version that includes this change.
 
+## [3.5.0]
+
+## [3.5.0]
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)